Rocket.Chat/Views/Cells/Subscription/SubscriptionCell.swift

```diff
@@ -16,6 +16,10 @@ final class SubscriptionCell: UITableViewCell {
     internal let labelReadTextColor = UIColor(rgb: 0x9AB1BF, alphaVal: 1)
     internal let labelUnreadTextColor = UIColor(rgb: 0xFFFFFF, alphaVal: 1)
 
+    internal let defaultBackgroundColor = UIColor.clear
+    internal let selectedBackgroundColor = UIColor(rgb: 0x0, alphaVal: 0.18)
+    internal let highlightedBackgroundColor = UIColor(rgb: 0x0, alphaVal: 0.27)
```

Comment:
@rafaelks I think these colors are highly related to this component, so I put these here.